src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/formatter/LocationFormatterTest.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.welcometoprison.formatter
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+class LocationFormatterTest {
+
+  var locationFormatter: LocationFormatter = LocationFormatter()
+
+  @Test
+  fun `When value is RECP formatter should return 'Reception'`() {
+    Assertions.assertEquals(locationFormatter.format("RECP"), "Reception")
```

Comment:
meh. we generally statically insert `Assertions`

Reply:
sorted